sdk/identity/ci.yml

```diff
@@ -74,9 +74,10 @@ extends:
           CalledFromClient: true
           ServiceDirectory: identity
           SupportedClouds: 'Public,UsGov,China'
-          UseFederatedAuth: false
           EnvVars:
-            SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+            ARM_OIDC_TOKEN: $(ARM_OIDC_TOKEN)
```

Comment:
In order to get these variables I think you will need to plumb the PersistOidcToken token open similar to what @benbp is doing in js at https://github.com/Azure/azure-sdk-for-js/pull/31335

Reply:
Yep, and we should remove usage of ARM_CLIENT_ID and ARM_TENANT_ID altogether, since these variables are already set as <service dir>_*_ID